FrontEnd/SignUpPage/src/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -0,0 +1,267 @@
+import { useForm } from 'react-hook-form';
+import styles from './SignUpFormContent.module.css'
+
+function SignUpFormContentComponent() {
+
+    const errorMessageTemplates = {
+        'required': 'Обов’язкове поле',
+        'email': 'invalid email, should contain @ and .xx',
+        'password': 'Пароль не відповідає вимогам',
+        'confirmPassword': "Паролі не збігаються",
+        'checkboxes': 'Будь ласка, оберіть кого ви представляєте'
+    }
+
+    const emailPattern = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+
+    const passwordPattern = /^(?=.*[0-9])(?=.*[a-zA-Z])[a-zA-Z0-9]{8,20}$/;
+
+
+    const {
+        register,unregister,
+        handleSubmit,
+        formState: { isValid, isSubmitted, errors },
+        watch, getValues, setValue
```

Comment:
This code looks unformatted. I would reccomed using some autoformatting tools like [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) for VSCode to format it.

Not in context of this story but you may consider using [esling](https://eslint.org/) or similar to keep your code clean.